scripts/debian/start_docker_debian_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -x
+set -v
+echo "Starting start_docker.sh"
+TEST_NAME="DebianTest"
+TEST_DIR=/tmp/debian-installer-test
```

Comment:
can we do a `mkdir` with at-exit `trap rm -rf` pattern like we have in some of the other build scripts? see build_deb.sh

Reply:
ok, that is cleaner, thanks.